include/glow/Graph/Graph.h

```diff
@@ -222,12 +223,20 @@ class Function final : public Named {
   /// A reference to the owner of the function.
   Module *parent_;
 
+  /// The log context associated with this function.
```

Comment:
Why do functions have log contexts? Shouldn't this belong elsewhere?

Reply:
Yeah I'm a bit confused -- @ZchiPitt  I thought you removed const from `CompilationContext` everywhere to then add the `LogContext` there?

Reply:
After consideration, we think it's better to move the LogContext into Function instead of CompilationContext, such that we can avoid touching many places to pass the the CompilationContext into these optimization phases.